fiftyone/core/map/mapper.py

```
@@ -0,0 +1,253 @@
+    if first_sample is None:
+        raise ValueError("Sample collection is empty")
+
+    # make a copy outside of the db
+    sample_copy = first_sample.copy()
+
+    # run the map function on just the copy
+    # if it returns a Sample object
+    if isinstance(map_fcn(sample_copy), fos.Sample):
+        return True
+
+    return False
```

Ruff (0.8.2)

Return the condition `bool(isinstance(map_fcn(sample_copy), fos.Sample))` directly

Replace with `return bool(isinstance(map_fcn(sample_copy), fos.Sample))`

(SIM103)